src/components/PermissionSettingsDemo/RoleManagementSettings.js

```
@@ -0,0 +1,120 @@
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import Badge from '../ProPreviews/common/Badge';
+
+const RoleManagementSettings = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)